src/index.js

```diff
@@ -74,6 +74,15 @@ const unexpectedEnzyme = {
       }
     );
 
+    childExpect.exportAssertion(
+      ['<ReactWrapper> [not] to exist', '<ReactWrapper> [not] to be present'],
+      (expect, reactWrapper) => {
+        expect.errorMode = 'bubble';
+
+        return expect(reactWrapper.exists(), '[not] to be true');
```

Comment:
How should we diff here, so when it fails it renders the reactWrapper?

Reply:
That is a good question. I don't think we have enough information here to do something useful. When is this even actually useful, don't you always want to use contain. (there are probably some edge cases.)